Python/Python.sublime-syntax

```diff
@@ -303,6 +315,17 @@ contexts:
           pop: true
         - include: expression-in-a-statement
 
+  case-body:
+    - meta_scope: meta.statement.structural-pattern.case.python
+    - include: line-continuation-or-pop
+    - match: \b(as)\b
+      scope: keyword.control.structural-pattern.case.as.python
+      set: case-as
+    - match: ':(?!=)'
+      scope: punctuation.section.block.structural-pattern.case.python
+      pop: true
+    - include: expression-in-a-statement
```

Comment:
As far as I can tell, this doesn't follow `as`-matching into structured matches, i.e. `case [0, ('+'|'-') as op, right]:` or `case [(int() | float()) as value]:`.

https://www.python.org/dev/peps/pep-0635/#as-patterns

Reply:
The easy way, was to add a context highlighting `as NAME` to `expressions-common` accepting it being highlighted as valid expression in inadequate positions.

The hard way is to add a full set of `case-pattern` contexts duplicating all group, tuple, list, set and most expression contexts just to add `as NAME` highlighting and probably exclude some contexts not required within this scope.